core/pouch/migrations.js

```diff
@@ -216,6 +216,31 @@ const migrations /*: Migration[] */ = [
         return doc
       })
     }
+  },
+  {
+    baseSchemaVersion: 7,
+    targetSchemaVersion: 8,
+    description: 'Use generated ids instead of string based on path',
+    affectedDocs: (docs /*: Metadata[] */) /*: Metadata[] */ => {
+      return docs.filter(doc => !doc._deleted)
+    },
+    run: (docs /*: Metadata[] */) /*: Metadata[] */ => {
+      return docs.map(doc => {
+        if (doc.remote) {
+          // update remote with main object values?
```

Comment:
It looks like you started something, but forget it after.

Reply:
ARf yes, I completely forgot the migration.